config/plugins/isecl/.key-server.conf

```diff
@@ -0,0 +1,9 @@
+# NOTE: update the below variables before deploying using
```

Comment:
Here too it would be better for `make deploy` to fail rather than using wrong values.

Reply:
This requires changes to how we use `deploy` and `deploy-manifests` targets. Currently `deploy-manifests` also uses `make deploy` to generate the manifest files. We cannot have these values set at the time of manifest generation hence cannot enforce. These values are only known when deploying the controller on to cluster.